cmd/oci-create-runtime-bundle/main.go

```diff
@@ -112,29 +102,26 @@ func (v *bundleCmd) Run(cmd *cobra.Command, args []string) {
 	logrus.SetLevel(logLevel)
 
 	if v.version {
-		v.stdout.Printf("commit: %s", gitCommit)
-		v.stdout.Printf("spec: %s", specs.Version)
+		logrus.Infof("commit: %s", gitCommit)
+		logrus.Infof("spec: %s", specs.Version)
```

Comment:
These are [supposed to go to stdout][1], so we should leave them alone.

[1]: https://github.com/opencontainers/image-tools/pull/9#discussion_r79332159